### ISSUE TITLE
Look at Placement of Adverts (see issue #612)

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#1212]
E3 U3 S6 C6: P18

**Comment by DeniseColbert** [comment]
Unsure what's needed here, #612 is deployed

**Comment by PatReynolds** [comment]
Issue #612 is in Ready for Testing, and this isn't about that issue , but the images there raise questions about the placement of adverts.

In balancing need for $$$ and UX, I'd like to suggest removing the second advert on the detailed result page, but implementing a sticky advert at the bottom here.  

On On the Records pages, it would be good for legibility for the tables to take up the normal "2/3" (I know it is not 2/3!) and there to be a side advert.

What do others think?

**Comment by DeniseColbert** [comment]
Sorry, I meant deployed to Test3. 

Thanks for the extra info, I take it that it was felt that the extra level of buttons in #612 made the layout unpleasant. I think your suggestions sound like a very good solution.

**Comment by PatReynolds** [comment]
Yes, the extra layer of buttons, while excellent, and the placement above/below the census information rather together will help people remember which 'layer' things are in, that comes at the expense of 'Show me my Great Uncle!!!'

**Comment by Captainkirkdawson** [comment]
If we move the ad to the bottom then 3 separate action lines is poor; I would suggest that the printing option go with the dwelling header since we print the dwelling and its contents. The citation can then go with the individuals header. This then gives us 2 nicely separated action header

**Comment by Captainkirkdawson** [comment]
@PatReynolds The problem with putting a sticky add at the bottom could be the fact that we have the Donate box etc under the individuals

**Comment by PatReynolds** [comment]
There is lots of scroll room below the invitation to donate (and  Top of Page and Report a Problem buttons),  _However_ there would need to be more black space under the copyright statement.

**Comment by ghost** [comment]
Remember Top of Page and Report a Problem buttons are not there for researchers

**Comment by PatReynolds** [comment]
For information, earnings last month
freecen_728x90 (top banner) £90.03
right_side_300x600 (right rectangle) £1.17

FC1 search £17.08
FC1 homepage £5.97

FreeREG comparisons are here: https://docs.google.com/spreadsheets/d/1i2YklMJScm3DGqnjf2O1GXX-DBPx0uKAVFF3PxiEMbQ/edit?usp=sharing 

Personally would prefer clean pages (i.e. no adverts apart from banner at top and rectangle at site - where there is a side), but with an easily dismissed sticky at the bottom.
